src/renderer/template.ts

```diff
@@ -13,7 +13,14 @@ export async function renderTemplate(filepath: string, context: TemplateContext)
   if (!isJsFile(filepath)) {
     return undefined;
   }
-  const data = await importComponent(filepath, context);
+
+  let data = undefined;
+  try {
+    data = await importComponent(filepath, context);
+  } catch(err) {
+    console.error(err);
```

Comment:
Here I don't know if we should throw error, or only log it. Any suggestions?

Reply:
I don't think it should be logged at all but just thrown so later in the generator we can catch it and log properly